onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -194,6 +181,13 @@ def run_test_output_match(
             f"Type constraints: {onnx_function.op_schema.type_constraints}"
         )
 
+    # Obtain the tolerance for the op if specified
+    if (tolerance := torchlib_op_info.tolerance.get(dtype)) is not None:
+        rtol, atol = tolerance
+    else:
+        rtol = None
```

Comment:
What is meaning of None? Please add comment :)

Reply:
None makes PyTorch to use the default tolerance for the data type. I will add a comment

Reply:
Done